static/etc/apt/auth.conf.d/01-fips.conf.example

```diff
@@ -0,0 +1,3 @@
+# replace the <LOGIN> and <PASSWORD> with your own private access
```

Comment:
Ah, I did not see, it is in static/. Please move this file in the root of the project. We should copy it in and remove it before packing.

Reply:
added, nice catch!